dynamic-plugins/imports/package.json

```diff
@@ -26,6 +26,7 @@
     "@janus-idp/backstage-plugin-ocm": "3.6.1",
     "@janus-idp/backstage-plugin-ocm-backend": "3.4.12",
     "@janus-idp/backstage-plugin-quay": "1.5.2",
+    "@janus-idp/backstage-plugin-rbac": "1.11.1",
```

Comment:
I think this should be 1.13.0.

Reply:
@gashcrumb Thanks for the review. Good catch. I have bumped up the version and tested locally. PR updated.